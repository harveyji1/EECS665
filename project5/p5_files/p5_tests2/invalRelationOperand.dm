main : () void {
    a : bool = true;
    b : bool;
    b =	a < false; 
}