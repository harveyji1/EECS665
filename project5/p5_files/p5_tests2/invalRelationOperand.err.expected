FATAL [4,9]-[4,10]: Relational operator applied to non-numeric operand
FATAL [4,13]-[4,18]: Relational operator applied to non-numeric operand
Type Analysis Failed
